metakb/harvesters/pmkb.py

```diff
@@ -0,0 +1,249 @@
+        newest_filename = sorted(files, reverse=True)[0]   # get most recent
+        infile = open(newest_filename, 'r')
+        data = pd.read_csv(infile, na_filter=False)
+        data.columns = ['gene', 'tumor_types', 'tissue_types', 'variants',
+                        'tier', 'interpretation', 'citations']
+        for col in ['variants', 'tumor_types', 'tissue_types', 'citations']:
+            data[col] = data[col].apply(lambda r: r.split('|'))
+            data[col] = data[col].apply(lambda r: [i.strip() for i in r])
+        for col in ['gene', 'interpretation']:
+            data[col] = data[col].apply(lambda r: r.strip())
+        return data
+
+    def _download_csv(self):
+        """Download source data from PMKB server. Doesn't return anything,
+        but saves file in the location designated by the instance's data_dir
+        attribute.
+        """
+        PMKB_URL = "https://pmkb.weill.cornell.edu/therapies/downloadCSV.csv"
+        logger.info("Downloading PMKB source CSV...")
+        response = requests.get(PMKB_URL, stream=True)
+        if response.status_code == 200:
+            fname = ''
+            if "Content-Disposition" in response.headers.keys():
+                fname = re.findall("filename=(.+)",
+                                   response.headers["Content-Disposition"])[0]
+                fname = fname.strip('\"')
+            else:
+                fname = PMKB_URL.split("/")[-1]
+            with open(self._data_dir / fname, 'wb') as f:
+                f.write(response.content)
+            logger.info("PMKB source CSV download successful.")
+        else:
+            logger.error(f"PMKB source download failed with status code: {response.status_code}")  # noqa: E501
+            raise FileDownloadException("PMKB source download failed")
+
+    def _build_genes(self, data: pd.DataFrame) -> List:
+        """Build list of genes.
+
+        :param DataFrame data: PMKB input data formatted as a Pandas DataFrame.
+        :return: completed List of gene items.
+        :rtype: List
+        """
+        genes = list()
+        genes_grouped = data[['gene', 'variants']].groupby('gene',
+                                                           as_index=False)
+        for (gene, group) in genes_grouped:
+            # flatten variant lists
+            var_series = group['variants'].apply(pd.Series) \
+                .stack().reset_index(drop=True)
+            # get counts for variants
+            var_counts = pd.Series([v for v in var_series if v != ''],
+                                   dtype=str).value_counts()
+            genes.append({
+                'type': 'gene',
+                'name': gene,
+                'variants': [
+                    {
+                        'name': name,
+                        'evidence_count': count,
+                    }
+                    for name, count in var_counts.iteritems()
+                ]
+            })
```

Comment:
it seems PMKB doesn't provide additional gene-specific data, same as MOA. Alex suggested to drop the gene entry, we might wanna consider the same for PMKB? 
check: https://github.com/cancervariants/metakb/pull/41#discussion_r555359648